<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetRendering</class>
 <widget class="QWidget" name="WidgetRendering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rendering</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Rendering">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
   </property>
   <property name="title">
    <string>Rendering</string>
   </property>
   <widget class="QCheckBox" name="checkBox_DrawFaceNormals">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Face Normals</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DrawEdge">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DrawTriangle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Triangle</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DrawPoint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Point</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_EdgeWidth">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Widthï¼š</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EdgeWidth">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Status</string>
   </property>
   <widget class="QPushButton" name="pushButton_OriginMesh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin Mesh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ResultMesh">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Result Mesh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NoisyMesh">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Noisy Mesh</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
